File: Sources/front/src/Api/Models/Entities/ProfileDto.ts
export interface ProfileDto {
  firstName?: string;
  lastName?: string;
  dateOfBirth?: string;
  gender?: "Male" | "Female" | "Other";
  height?: number;
  //heightUnit?: "Cm" | "In";
  weight?: number;
  //weightUnit?: "Kg" | "Lb";
  //avatarUrl?: string;
}

File: Sources/bff/E-Manger-Mieux.Web/Models/ProfileDto.cs
namespace Models;

using Models.Enums;

public class ProfileDto
{    
    public string? FirstName { get; set; }

    public string? LastName { get; set; }

    public DateTimeOffset? DateOfBirth { get; set; }

    public GenderType? Gender { get; set; }

    public decimal? Height { get; set; } 

    public decimal? Weight { get; set; } 

    public string? AvatarUrl { get; set; }
}
